src/v2/controllers/FormController.js

```diff
@@ -153,39 +152,29 @@ export default Controller.extend({
   handleSaveForm(model) {
     const formName = model.get('formName');
 
-    const idx = this.options.appState.get('idx');
-    if (!idx['neededToProceed'].find(item => item.name === formName)) {
-      this.options.settings.callGlobalError(`Cannot find http action for "${formName}".`);
-      this.showFormErrors(this.formView.model, 'Cannot find action to proceed.');
-      return;
-    }
-
+    // Toggle Form saving status (e.g. disabling save button, etc)
     this.toggleFormButtonState(true);
     model.trigger('request');
```

Comment:
nit: Should these be handled via the controller for BaseForm/View? Seems like more of a form-level concern to me! 😄

Reply:
Good point! I'll make sure it's got addressed at my next PR.

Reply:
The problem was there are several cases  `saveForm` was triggered on `appState` directly instead of `form.trigger('save')`, e.g. polling. The nuance is whether want to clean up error (`form.trigger(save)` does clean up form errors).
Need to dig into further and may require more refactoring, like move `handeSaveForm` into BaseForm.